azkaban-common/src/main/java/azkaban/utils/PropsUtils.java

```diff
@@ -330,17 +330,10 @@ public static String toJSONString(Props props, boolean localOnly) {
     return map;
   }
 
-  @SuppressWarnings("unchecked")
-  public static Props fromJSONString(String json) {
-    try {
-      Map<String, String> obj =
-          (Map<String, String>) JSONUtils.parseJSONFromString(json);
-      Props props = new Props(null, obj);
-      return props;
-    } catch (IOException e) {
-      logger.error("Encountered error during parsing project properties json. ", e);
-      return null;
-    }
+  public static Props fromJSONString(String json) throws IOException {
+    Map<String, String> obj = (Map<String, String>) JSONUtils.parseJSONFromString(json);
+    Props props = new Props(null, obj);
```

Comment:
Yes.

I don't have a strong opinion either way. 

I wanted to minimize my changes.